### PR TITLE
[Feature][Connector-V2-e2e][LocalFile] Add local file connector e2e

### LINKED ISSUE #3159: [Feature][e2e] Add local file connector e2e tests
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/incubator-seatunnel/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

Add local file connector e2e tests

### Usage Scenario

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by TyrantLucifer**
## Purpose of this pull request

close #3159 

## Check list

* [x] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-file-local-e2e/src/test/java/org/apache/seatunnel/e2e/connector/file/local/LocalFileIT.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    @TestTemplate
+    public void testLocalFileWrite(TestContainer container) throws IOException, InterruptedException {
+        Container.ExecResult textResult = container.executeJob("/text/fake_to_local_file_text.conf");
+        Assertions.assertEquals(0, textResult.getExitCode());
+        Container.ExecResult jsonResult = container.executeJob("/json/fake_to_local_file_json.conf");
+        Assertions.assertEquals(0, jsonResult.getExitCode());
+        Container.ExecResult orcResult = container.executeJob("/orc/fake_to_local_file_orc.conf");
+        Assertions.assertEquals(0, orcResult.getExitCode());
+        Container.ExecResult parquetResult = container.executeJob("/parquet/fake_to_local_file_parquet.conf");
```
Comment (hailin0) [inline seatunnel-e2e/seatunnel-connector-v2-e2e/connector-file-local-e2e/src/test/java/org/apache/seatunnel/e2e/connector/file/local/LocalFileIT.java]: Add `local_file_xxx_to_assert.conf` validate row count & field types?


Or

you can read sink data check rows & fields

Comment (TyrantLucifer) [reply seatunnel-e2e/seatunnel-connector-v2-e2e/connector-file-local-e2e/src/test/java/org/apache/seatunnel/e2e/connector/file/local/LocalFileIT.java]: Yup, this pull request is not completed. It's also a draft pr.

**Review by EricJoy2048 — APPROVED** [review]
LGTM

**Review by ashulin — APPROVED** [review]
+1

**Pull request merged**